README.md

```diff
@@ -1,46 +1,73 @@
 # .NET SDK for Lucidtech AI Services
 
+## Documentation
+
+[Link to docs](https://docs.lucidtech.ai/dotnet/v1/index.html)
+
+Create documents by using doxygen.
+Download the latest and greatest version of [doxygen](https://github.com/doxygen/doxygen.git).
+```bash
+$ doxygen documentation.conf
+```
+
 ## Installation
 
 ### NuGet
 
-```
-$ nuget install Lucidtech.LAS.Receipt
+```bash
+$ nuget install Lucidtech.Las
 ```
 
 ## Usage
-Instantiate the Lucidtech.LAS.Client with your API key:
 
-``` C#
-using Lucidtech.LAS;
+### Preconditions
 
-// ...
+- Documents must be in upright position
+- Only one receipt or invoice per document is supported
+- Supported file formats are: jpeg, pdf
+- Lucidtech.Las requires compatibility with .NET standard 4.7
 
-string apiKey = "...";
-var client = new Client(apiKey);
-```
+### Quick start
 
-To scan a receipt, call the API either with a URL to an image file:
 ```C#
-string receiptUrl = "...";
-List<Detection> detections = client.ScanReceipt(receiptUrl);
+using Lucidtech.Las;
+
+# Run inference and create prediction on document 
+using namespace Lucidtech.Las;
+ApiClient apiClient = new ApiClient("<api endpoint>");
+Prediction response = apiClient.Predict(documentPath: "document.pdf", modelName: "invoice");
+Console.WriteLine(response.ToJsonString(Formatting.Indented));
+
+# Send feedback to the model.
+var feedback = new List<Dictionary<string, string>>()
+{ 
+    new Dictionary<string, string>(){{"label", "total_amount"},{"value", "54.50"}},
+    new Dictionary<string, string>(){{"label", "purchase_date"},{"value", "2007-07-30"}}
+};
+FeedbackResponse response = apiClient.SendFeedback(documentId: "<documentId>", feedback: feedback);
+Console.WriteLine(response.ToJsonString(Formatting.Indented));
+
+# Revoke consent and deleting all documents associated with consentId.
+ApiClient apiClient = new ApiClient("<endpoint>"); \n
+RevokeResponse response = apiClient.RevokeConsent(consentId: "<consentId>");
+Console.WriteLine(response.ToJsonString(Formatting.Indented));
+ 
 ```
 
-or with a System.IO.Stream object:
+## Contributing
 
-```C#
-FileStream stream = new FileStream(/* ... */); 
-List<Detection> detections = client.ScanReceipt(stream);
+### Prerequisites
+Download the latest and greatest version of [MSBuild](https://github.com/Microsoft/msbuild).
+
+Download NUnit.Console version 3.9.0 or higher to run tests from command line.
+```bash
+$ nuget install NUnit.Console --version 3.9.0
 ```
 
-The detection class contains a label (e.g. total or date), a confidence estimation and the extracted value:
-```C#
-foreach (var detection in detections)
-{
-    Console.WriteLine(detection.Label);
-    Console.WriteLine(detection.Confidence);
-    Console.WriteLine(detection.Value);
-}
+### Run tests
+
+```bash
+$ msbuild Test/Test.csproj # Build 
```

Comment:
Running this command produces:
Microsoft (R) Build Engine version 16.0.0.0 for Mono
Copyright (C) Microsoft Corporation. All rights reserved.

Build started 3/22/2019 12:48:00 PM.
/home/august/dev/LucidtechAI/las-sdk-net/Test/Test.csproj(3,3): warning MSB4011: "/usr/lib/mono/xbuild/15.0/Microsoft.Common.props" cannot be imported again. It was already imported at "/usr/lib/mono/msbuild/15.0/bin/Sdks/Microsoft.NET.Sdk/Sdk/Sdk.props (33,3)". This is most likely a build authoring error. This subsequent import will be ignored. 
/usr/lib/mono/msbuild/15.0/bin/Sdks/Microsoft.NET.Sdk/Sdk/Sdk.targets(41,3): warning MSB4011: "/usr/lib/mono/msbuild/15.0/bin/Microsoft.CSharp.targets" cannot be imported again. It was already imported at "/home/august/dev/LucidtechAI/las-sdk-net/Test/Test.csproj (79,3)". This is most likely a build authoring error. This subsequent import will be ignored. 
Project "/home/august/dev/LucidtechAI/las-sdk-net/Test/Test.csproj" on node 1 (default targets).
/usr/lib/mono/msbuild/15.0/bin/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.DefaultItems.targets(245,5): error : Duplicate 'Compile' items were included. The .NET SDK includes 'Compile' items from your project directory by default. You can either remove these items from your project file, or set the 'EnableDefaultCompileItems' property to 'false' if you want to explicitly include them in your project file. For more information, see https://aka.ms/sdkimplicititems. The duplicate items were: 'Properties/AssemblyInfo.cs'; 'Tests.cs' [/home/august/dev/LucidtechAI/las-sdk-net/Test/Test.csproj]
Done Building Project "/home/august/dev/LucidtechAI/las-sdk-net/Test/Test.csproj" (default targets) -- FAILED.

Build FAILED.

  /home/august/dev/LucidtechAI/las-sdk-net/Test/Test.csproj(3,3): warning MSB4011: "/usr/lib/mono/xbuild/15.0/Microsoft.Common.props" cannot be imported again. It was already imported at "/usr/lib/mono/msbuild/15.0/bin/Sdks/Microsoft.NET.Sdk/Sdk/Sdk.props (33,3)". This is most likely a build authoring error. This subsequent import will be ignored. 
  /usr/lib/mono/msbuild/15.0/bin/Sdks/Microsoft.NET.Sdk/Sdk/Sdk.targets(41,3): warning MSB4011: "/usr/lib/mono/msbuild/15.0/bin/Microsoft.CSharp.targets" cannot be imported again. It was already imported at "/home/august/dev/LucidtechAI/las-sdk-net/Test/Test.csproj (79,3)". This is most likely a build authoring error. This subsequent import will be ignored. 


"/home/august/dev/LucidtechAI/las-sdk-net/Test/Test.csproj" (default target) (1) ->
(CheckForDuplicateItems target) -> 
  /usr/lib/mono/msbuild/15.0/bin/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.DefaultItems.targets(245,5): error : Duplicate 'Compile' items were included. The .NET SDK includes 'Compile' items from your project directory by default. You can either remove these items from your project file, or set the 'EnableDefaultCompileItems' property to 'false' if you want to explicitly include them in your project file. For more information, see https://aka.ms/sdkimplicititems. The duplicate items were: 'Properties/AssemblyInfo.cs'; 'Tests.cs' [/home/august/dev/LucidtechAI/las-sdk-net/Test/Test.csproj]

    2 Warning(s)
    1 Error(s)

Time Elapsed 00:00:00.53

Arch Linux:
4.20.1-arch1-1-ARCH